### PR TITLE
More for the community

### LINKED ISSUE #156: Highlight contributors in README
To make all contributors more visible we should highlight them in our README.rst. With this they will also appear prominently on the pip webpage. I see two options here

# 1. Avatars only
The easiest way is to use [contributors-img](https://contrib.rocks/preview). This will generate picture that we can use and linked to our contributor page.

<a href="https://github.com/lab-cosmo/equistore/graphs/contributors">
  
</a>

# 2. Avatars and names (and emojis)

Another way would to also show the names. This could be done in an automatic way using [all-contributors](https://github.com/all-contributors/app) project. I don't like the emojis but we can maybe disable them.

Screenshot 2023-03-02 at 16 56 22

**all-contributor**s has a lot of options and it takes the reference from a Contributor file.

I like the simple solution with the avatars, mostly because there is nothing to configure :-).

### PR DESCRIPTION
**Pull request by PicoCentauri**
Fixes #156

You can find the rendered version [here](https://github.com/lab-cosmo/equistore/tree/visible_community).

Also, I updated and converted the `CODE_OF_CONDUCT.rst` (coc) into a markdown file. A markdown version is not problematic  because we do not use the coc file in our docs. Also, with the conversion GitHub should recognizes the file and should show this in our projects sidebar. At least it worked if I pushed this to the master branch of my [fork](https://github.com/PicoCentauri/equistore):

Screenshot 2023-03-03 at 11 02 47

----
:books: Documentation preview :books:: https://equistore--157.org.readthedocs.build/en/157/

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+mail.
```
Comment (Luthaf) [inline CODE_OF_CONDUCT.md:63]: should we put some contact info here?

Comment (PicoCentauri) [reply CODE_OF_CONDUCT.md:63]: Yes, good idea. I wanted to ask this during the creation but I forgot. Do we have a adress for such cases?

Comment (Luthaf) [reply CODE_OF_CONDUCT.md:63]: Not really =/ We could put one of our email address, or the lab webpage, or @ceriottm email address?

Comment (PicoCentauri) [reply CODE_OF_CONDUCT.md:63]: Maybe @ceriottm is not a bad idea, if he agrees and wants to become the community leader. I would also be fine with mine though...

Comment (Luthaf) [reply CODE_OF_CONDUCT.md:63]: Let's go with yours for now, we can always update it later =)

Comment (PicoCentauri) [reply CODE_OF_CONDUCT.md]: Sure. I put my mail in :-)

**Review by Luthaf — APPROVED** [review]

**Pull request merged**